pkg/apb/client.go

```diff
@@ -185,16 +171,13 @@ func (c *Client) PullImage(imageName string) error {
 	return nil
 }
 
-func (c *Client) refreshLoginToken(clusterConfig ClusterConfig) error {
-	c.log.Debug("Refreshing login token...")
-	c.log.Debug("target: [ %s ]", clusterConfig.Target)
-	c.log.Debug("user: [ %s ]", clusterConfig.User)
-	c.log.Debug("password:[ %s ]", clusterConfig.Password)
+func (c *Client) Login() error {
+	c.log.Debug("Logging in to openshift...")
 
 	output, err := runCommand(
-		"oc", "login", "--insecure-skip-tls-verify", clusterConfig.Target,
-		"-u", clusterConfig.User,
-		"-p", clusterConfig.Password,
+    "oc", "login", "https://kubernetes.default",
+    "--certificate-authority", "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
+    "--token", "$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
```

Comment:
My current plan was to get to TP with that assumption and then go back and add in the logic that will let us run well in and out of a cluster. Talked with @eriknelson and he seemed to agree. AFAICT you can't really run our official APBs without the catalog at this point, so I didn't think it was a huge compromise.